src/main/java/co/inventorsoft/dbaccessjpa/service/EmailService.java

```diff
@@ -0,0 +1,49 @@
+package co.inventorsoft.dbaccessjpa.service;
+
+import co.inventorsoft.dbaccessjpa.model.Email;
+import co.inventorsoft.dbaccessjpa.repository.EmailRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.time.LocalDateTime;
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Service
+public class EmailService {
+
+    private final EmailRepository emailRepository;
+
+    public EmailService(EmailRepository emailRepository) {
+        this.emailRepository = emailRepository;
+    }
+
+    @Transactional
+    public void saveEmail(Email email) {
+        emailRepository.save(email);
+    }
+
+    @Transactional
+    public List<Email> findAll() {
+        return emailRepository.findAll();
+    }
+
+    @Transactional
+    public Email findById(Long id){
```

Comment:
1. Please format code properly.
2. For read operations read transaction is preffered one.